react/features/base/dialog/components/Dialog.web.js

```diff
@@ -31,6 +27,11 @@ class Dialog extends AbstractDialog {
         isModal: React.PropTypes.bool,
 
         /**
+         * Disables rendering of the submit button.
+         */
+        submitDisabled: React.PropTypes.bool,
+
```

Comment:
We were using wrong property and I have fixed this in another PR (https://github.com/jitsi/jitsi-meet/pull/1632). We have a property okDisabled defined in the Abstract dialog and it was only speaker stats that was using submitDisabled, so I got rid of it. Maybe we can merge this PR and then I fill fix the merge conflict in my PR with fixing this.